README.md

```diff
@@ -192,12 +192,17 @@ var client = new Client(config);
 var posPaymentLocalApi = new PosPaymentLocalApi(client);
```

Comment:
Can you please refactor the examples in the readme to use the non-deprecated methods?(or add the step in the new ticket)
```suggestion
var terminalLocalApi = new TerminalLocalApi(client);
```